examples/README.MD

```diff
@@ -0,0 +1,353 @@
+</td>
+</tr>
+<tr>
+<td>
+
+[apfl_example](apfl_example)
+</td>
+<td>
+This implements an example of training a model using the pFL approach, APFL. Server-side modifications are minimal, but the client-side training modifications are handled by the appropriate parent class.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[basic_example](basic_example)
+</td>
+<td>
+This is one of the simplest FL examples. It demonstrates federally training a model with basic FedAvg. It also provides an example of using server-side checkpointing and evaluating on a client-side test dataset in every round.
+
+Dataset: CIFAR 10
+</td>
+</tr>
+<tr>
+<td>
+
+[ditto_example](ditto_example)
+</td>
+<td>
+This example implements the pFL method Ditto to train individual client models on a heterogeneous version of MNIST. Ditto is one of a small set of methods that actually requires two optimizers for training.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[dp_fed_examples](dp_fed_examples)
+</td>
+<td>
+The examples in this folder are basic implementations of differentially private (DP) FL training. There are two levels of DP guarantees. Client-level DP refers to guarantees protecting <b>client</b> participation in FL training. Instance-level DP refers to guarantees protecting <b>datapoint</b> participation in FL training. Instance-level DP is classically achieved using DP-SGD on the client-side. The client-level examples include adaptive clipping implementations.
+
+Dataset: CIFAR-10, Medical EHR
+</td>
+</tr>
+<tr>
+<td>
+
+[dp_scaffold_example](dp_scaffold_example)
+</td>
+<td>
+This implementation is an example of using the global FL SCAFFOLD method with instance-level differential privacy guarantees (DP-SGD on the client-side). It also leverages the "warm-start" approach to SCAFFOLD.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[dynamic_layer_exchange_example](dynamic_layer_exchange_example)
+</td>
+<td>
+This example demonstrates how one might use the dynamic <b>layer</b> exchanger. This exchanger allows a user to adaptively exchange only a subset of layers that meet certain criterion (set by the user). Facilitates communication efficiency at the cost of some performance.
+
+Dataset: CIFAR-10
+</td>
+</tr>
+<tr>
+<td>
+
+[ensemble_example](ensemble_example)
+</td>
+<td>
+Training an ensemble of models is common in many applications. This example provides an implementation of training multiple models individually and simultaneously while measuring ensembled performance at prediction time.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[feature_alignment_example](feature_alignment_example)
+</td>
+<td>
+Provides an implementation of automatic feature alignment for tabular datasets. When data is represented in dataframes, this implementation automatically unifies the shared features and feature spaces to prepare the datasets for FL prior to training
+
+Dataset: Mimic3 Tabular Dataset
```

Comment:
nit: The rest of the datasets that are acronyms are capitalized

Reply:
👍